tutorials/resources/utils/weights_translation.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict
+import tensorflow as tf
+import numpy as np
+
+
+ln_patch = False
+
+
+def weight_translation(keras_name, pytorch_weights_dict, layer):
```

Comment:
Since it seems as a generic code, I suggest to add a description to each function
I'd also rename it to keras_to_pytorch_weight_translation or maybe the file name should be more informative